### DISCUSSION TITLE
Bicep Support

### DISCUSSION THREAD

**Discussion by vicperdana** [Azure/PSDocs.Azure#136] [Ideas]
Just starting a Bicep discussion, keen to hear some ideas how we can include these in the upcoming release.

**Comment by BertusV** [comment]
It would be great to have Bicep support to this excellent resource.

Here is what I would expect:
Documentation to reflect Bicep examples.
An option to accept Bicep input (or better automatic based on file extension)
An option to either generate the parameter example in JSON or BICEPPARAM.

I'm only just starting with PSDocs so I'll add more information as I go.

**Reply by vicperdana** [reply]
Thanks @BertusV this helps our planning.  Do you use PSDocs as part of your pipeline deployments?  Are you aware of [psdocs.vscode](https://marketplace.visualstudio.com/items?itemName=vicperdana.PSDocs-vscode-preview) extension?

**Reply by BertusV** [reply]
Hi @vicperdana , I only just started and would like to use PSDocs as part of my pipeline for IaC publishing Bicep templates to a Private Registry.  

Thanks for the heads up. I'm going to check out the VS Cone extension today!
